packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -670,8 +693,14 @@ export class Project extends ProjectBase {
       secondarySources: new cdk.Token(() => this.renderSecondarySources()),
       secondaryArtifacts: new cdk.Token(() => this.renderSecondaryArtifacts()),
       triggers: this.source.buildTriggers(),
+      vpcConfig: this._vpc ? {
+        vpcId: this._vpc.vpcId,
+        securityGroupIds: this._vpc_securityGroupIDs ? this._vpc_securityGroupIDs : undefined,
+        subnets: this._vpc_subnets ? this._vpc_subnets : undefined
+      } : undefined
```

Comment:
Don't we need to provide some permissions on the Project Role in the case that a VPC has been specified?